base/util.jl

```diff
@@ -383,7 +383,7 @@ macro timed(ex)
 end
 
 function fftw_vendor()
-    if Base.libfftw_name == "libmkl_rt"
+    if Base.libfftw_name == "libmkl_rt" || (is_windows() && Base.libfftw_name == "mkl_rt")
```

Comment:
I don't think the `is_windows()` check is strictly necessary, I doubt Intel would distribute a `mkl_rt.so` or `mkl_rt.dylib` but if they did we'd likely want to count it here.